layouts/partials/contribute/sigs.html

```diff
@@ -20,7 +20,7 @@
       SIG-Platform
     </a>
     <a href="https://github.com/o3de/sig-presentation" class="card sig-card">
```

Comment:
GitHub seems to correctly redirect this url, but we may as well change it to the correct one while we're at it :D
https://github.com/o3de/sig-graphics-audio